jni/events/baseaudioevent.cpp

```diff
@@ -513,3 +513,31 @@ void BaseAudioEvent::destroyBuffer()
         _buffer = 0;
     }
 }
+
+/* TO BE DEPRECATED */
+
```

Comment:
Keeping the old function names as this has been the event positioning API for the past 4 years... =/